hydroflow/src/lang/clear.rs

```diff
@@ -46,7 +47,9 @@ clear_impl!(BTreeMap<K, V>);
 clear_impl!(BTreeSet<T>);
 clear_impl!(BinaryHeap<T>);
 clear_impl!(HashMap<K, V>);
+clear_impl!(FxHashMap<K, V>);
 clear_impl!(HashSet<T>);
+clear_impl!(FxHashSet<T>);
```

Comment:
Would it work to just change the existing impl to add a hasher?
```rust
clear_impl!(HashMap<K, V, S>);
clear_impl!(HashSet<T, S>);
```

Reply:
ya it does work